scene/main/timer.cpp

```diff
@@ -108,7 +108,7 @@ bool Timer::has_autostart() const {
 
 void Timer::start(float p_time) {
 
-	ERR_FAIL_COND_MSG(!is_inside_tree(), "Timer was not added to the SceneTree!");
+	ERR_FAIL_COND_MSG(!is_inside_tree(), "Timer was not added to the SceneTree! (To fix: add to SceneTree or set autostart to true.)");
```

Comment:
I think it would be better if it was written as:
`"Timer was not added to the SceneTree. Either add it or set autostart to true."`

Reply:
Agreed. Made the change in the pull request branch.